jormungandr/src/settings/start/mod.rs

```diff
@@ -92,6 +83,45 @@ impl RawSettings {
             });
         }
 
+        // If the command line specifies log arguments, check that the arg
+        // is Some(output), else, skip.
+        let cmd_output = self.command_line.log_output.clone();
+
+        if let Some(output) = cmd_output {
+            let cmd_level = self.command_line.log_level;
+            let cmd_format = self.command_line.log_format;
+
+            // Read the log settings, and look if we already
+            // have this output configured
+            for entry in entries.iter_mut() {
+                // If the output is aleady configured:
+                if &entry.output == &output {
+                    // overwrite entry.level if cli_entry.level is Some(level)
+                    // overwrite entry.format if cli_entry.format is Some(format)
+                    let override_entry = LogSettingsEntry {
+                        level: cmd_level.unwrap_or(entry.level),
+                        format: cmd_format.unwrap_or(entry.format),
+                        output: output.clone(),
+                    };
+                    // log to info! that the output was overriden
+                    tracing::info!(
+                        "Log settings overriden from cli: {:?} replaced with {:?}",
+                        entry,
+                        override_entry
+                    );
```

Comment:
The trouble with this is that the subscribers are not initialized yet. Maybe it's best to add an `Option` member to `LogSettings` with these details, then, after the global subscriber has been set, check it and inform.

Reply:
Another problem is that the default entry is created in the code above if the config list is empty. In that case, we don't want a warning.